userguide/sharing.rst

```diff
@@ -928,6 +928,10 @@ provides more details for each configurable option.
    | Only Allow Guest Access        | checkbox      | ✓        | Requires :guilabel:`Allow guest access` to also be enabled. Forces guest access for all connections.                                                 |
    |                                |               |          |                                                                                                                                                      |
    +--------------------------------+---------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+   | Access Based Share Enumeration | checkbox      | ✓        | Restrict share visibility to Windows users with read or write access to the share, based on the current Windows Share ACL. Use Windows               |
```

Comment:
The last clause makes it a bit unclear. Does "Restrict share visibility to Windows users with current Windows Share ACL access of read or write." capture the correct meaning of this?

Also, not sure if Share should be capitalized in this or the next sentence.